### PR TITLE
Remove queryId  & ImageId in Pattern 

### PR DESCRIPTION
**Pull request by shail-mehta**
Removed query Id  & Image Id in Pattern as per Pattern Creation Guideline

**Files:**
hidden-sidebar.php
hidden-sidebar-writer.php

### CODE REVIEW

File: patterns/hidden-sidebar-writer.php [hunk 1]
```diff
@@ -8,7 +8,7 @@
 ?>
 <!-- wp:group {"style":{"spacing":{"blockGap":"2.5rem"}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch"}} -->
 <div class="wp-block-group"><!-- wp:group {"layout":{"type":"constrained"}} -->
-<div class="wp-block-group"><!-- wp:image {"id":60,"width":80,"height":80,"scale":"cover","sizeSlug":"large","linkDestination":"none"} -->
+<div class="wp-block-group"><!-- wp:image {"width":80,"height":80,"scale":"cover","sizeSlug":"large","linkDestination":"none"} -->
 <figure class="wp-block-image size-large is-resized"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/writer-avatar.webp" alt="" class="wp-image-60" style="object-fit:cover;width:80px;height:80px" width="80" height="80"/></figure>
```
Comment (MaggieCabrera) [inline patterns/hidden-sidebar-writer.php]: we need to remove the class as well or the block will be broken in the site editor

Comment (shail-mehta) [reply patterns/hidden-sidebar-writer.php]: @MaggieCabrera I have removed class also

**Comment by carolinan** [comment]
Hi,
These two images should be changed to the avatar block, and someone has already signed up to work on that issue, please see:
https://github.com/WordPress/twentytwentyfour/pull/314

Please update the pr to only remove the query id.

**Comment by shail-mehta** [comment]
@carolinan 

I have reverted images related changes & now only removed queryId

**Review by MaggieCabrera — APPROVED** [review]

**Pull request merged**